src/cargo-fmt/main.rs

```diff
@@ -198,12 +198,10 @@ fn convert_message_format_to_rustfmt_args(
             Ok(())
         }
         "human" => Ok(()),
-        _ => {
-            return Err(format!(
-                "invalid --message-format value: {}. Allowed values are: short|json|human",
-                message_format
-            ));
-        }
+        _ => Err(format!(
```

Comment:
warning: unneeded `return` statement
```
  --> src/lib.rs:43:13
   |
43 | /             return Err(format!(
44 | |                 "invalid --message-format value: {}. Allowed values are: short|json|human",
45 | |                 message_format
46 | |             ));
   | |_______________^
   |
```
   = note: `#[warn(clippy::needless_return)]` on by default
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_return